my_portfolio.html

```diff
@@ -1,13 +1,63 @@
 ﻿<!DOCTYPE html>
-
 <html lang="en">
+
 <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css" />
 </head>
+
 <body>
-    <h1>Hello world</h1>
+    <img src="icons/header-illustration-mobile.svg" id="header-img" />
+    <img src="icons/header-desktop.svg" id="header-img-desktop" />
+    <div id="nav-bar-ctn">
+        <div id="nav-bar">
+            <a>
+                <p>Portfolio</p>
+            </a>
+            <a>
+                <p>About</p>
+            </a>
+            <a>
+                <p>Contact</p>
+            </a>
+        </div>
+    </div>
+    <div id="toolbar">
+        <div id="menu-button">
+            <object data="icons/icon-menu.svg"></object>
+        </div>
+    </div>
+
+    <div id="headline">
+
+        <div id="primary-text">
+            <p id="title-1">Hey there. I'm Gumaro.</p>
+            <p id="title-2">I'm a software developer</p>
+        </div>
+        <div id="support-text">
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hesitate to contact me.
+            </p>
+        </div>
+        <div id="social-media">
+            <img src="icons/twitter.svg" />
+            <img src="icons/linkedin.svg" />
+            <img src="icons/medium.svg" />
+            <img src="icons/github.svg" />
+            <img src="icons/angellist.svg" />
+        </div>
+    </div>
+
+    <div id="down-btn-ctn">
+        <button type="button">
+            <img src="icons/union.svg" />
+        </button>
+    </div>
```

Comment:
- [ ] Kindly consider wrapping all this in a section tag because it represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.t represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.